app/assess/routes.py

```diff
@@ -289,16 +293,27 @@ def application(application_id):
     if flag:
         state.workflow_status = "FLAGGED"
         accounts = get_bulk_accounts_dict([flag.user_id])
-
+        
+    sub_criteria_status_completed = all_status_completed(state)
+    form = AssessmentCompleteForm()
+    if request.method=="POST":
+        requests.post(
+            Config.ASSESSMENT_UPDATE_STATUS.format(
+                application_id=application_id)
+            )
```

Comment:
Indentation is messed up here, could you install the pre-commit hooks so they run when making changes?




From the root of the project run:
```
pre-commit install 
```

Otherwise other people commits will fix the formatting and it will pollute their PRs with unrelated changes, this should also help with merge conflicts we'll be using the same format & have sorted imports, so less work for others when solving conflicts.

Reply:
I am alreday tiding up as we speak 😄

Reply:
👍 IF you install the hooks this gets done for you when committing